tests/integration_tests/monetization_api.test.ts

```diff
@@ -329,7 +329,7 @@
 
       const vungleInstance = new VungleInstance('Test_2', MonetizeEnums.AdUnits.RewardedVideo, true, '', '', 'TEST', { rate: 2.5 });
       const res: any = await ironsource.MonetizeAPI().addInstances(testAppKey, [vungleInstance]);
-      expect(res.rewardedVideo.Vungle).to.be.an('array').with.lengthOf(2);
+      expect(res.rewardedVideo.liftoffMonetize).to.be.an('array').with.lengthOf(2);
```

Comment:
The test description is not accurate anymore. The test is checking for the length of the array of a different instance than Vungle. Please update the test description to reflect the changes made.